metadata/src/test/java/org/apache/kafka/controller/BrokerHeartbeatTrackerTest.java

```diff
@@ -43,12 +43,11 @@ private static BrokerHeartbeatTracker newBrokerHeartbeatTracker() {
     private static final Set<BrokerIdAndEpoch> TEST_BROKERS;
 
     static {
-        Set<BrokerIdAndEpoch> brokers = new HashSet<>();
-        Arrays.asList(
-            new BrokerIdAndEpoch(0, 0L),
-            new BrokerIdAndEpoch(1, 100L),
-            new BrokerIdAndEpoch(2, 200L)
-        ).forEach(brokers::add);
+        Set<BrokerIdAndEpoch> brokers = new HashSet<>(Arrays.asList(
```

Comment:
Maybe we can leverage `Set.of`
```java
    private static final Set<BrokerIdAndEpoch> TEST_BROKERS = Set.of(
        new BrokerIdAndEpoch(0, 0L),
        new BrokerIdAndEpoch(1, 100L),
        new BrokerIdAndEpoch(2, 200L)
    );
```

Reply:
Yeah I kind of refrained myself from using `Set.of()`/`List.of()` to not cause too many changes but in this case it makes sense to use `Set.of()` and remove the ugly `static` block. Done, thanks!